assets/dev/js/editor/controls/date-time.js

```diff
@@ -10,6 +10,60 @@ export default class extends ControlBaseDataView {
 		this.ui.input.flatpickr( options );
 	}
 
+	events() {
+		return {
+			...super.events(),
+			'change @ui.input': 'onBaseInputChange',
+		};
+	}
+
+	onBaseInputChange() {
+		super.onBaseInputChange( ...arguments );
+		if ( this.model.attributes.datetime_verify ) {
+			const startDate = this.options.container.settings.get( this.model.attributes.datetime_verify.control_name );
+			if ( ! startDate ) {
+				return;
+			}
+
+			const endDate = this.ui.input[ 0 ].value;
+			const startDateTimestamp = new Date( startDate ).getTime();
+			const endDateTimestamp = new Date( endDate ).getTime();
+
+			switch ( this.model.attributes.datetime_verify.operator ) {
+				case '>=':
+					if ( startDateTimestamp >= endDateTimestamp ) {
+						this.ui.input[ 0 ].value = '';
+					}
+					break;
+				case '<=':
+					if ( startDateTimestamp <= endDateTimestamp ) {
+						this.ui.input[ 0 ].value = '';
+					}
+					break;
+				case '>':
+					if ( startDateTimestamp > endDateTimestamp ) {
+						this.ui.input[ 0 ].value = '';
+					}
+					break;
+				case '<':
+					if ( startDateTimestamp < endDateTimestamp ) {
+						this.ui.input[ 0 ].value = '';
+					}
+					break;
```

Comment:
maybe you can use `elementor.conditions.compare`:

```JS
// ... 
const startDateTimestamp = new Date( startDate ).getTime();
const endDateTimestamp = new Date( endDate ).getTime();
const operator = '...';

if ( ! elementor.conditions.compare( startDateTimestamp, endDateTimestamp ) ) {
    this.ui.input.val( '' );
}
```

Reply:
https://github.com/elementor/elementor/blob/dffd09ea02a36d4130dd310c5e2b5b0a4fc615b3/assets/dev/js/editor/utils/conditions.js

Reply:
Nice one, Done.